src/components/atoms/icon-button/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import Icon from '_atoms/icon'
+
+import styles from './styles.css'
+
+const IconButton = ({ iconURL, onClick, className, classNameIcon }) => (
+  <button className={classNames(className, styles.iconButton)} onClick={onClick}>
```

Comment:
use your Button component instead of html button.

Reply:
Yeah, but the button component is already created in another branch. We need to merge all component in a specific branch to avoid this issue :/